src/cow-react/modules/trade/hooks/setupTradeState/useSetupTradeState.ts

```diff
@@ -10,19 +10,25 @@ import usePrevious from 'hooks/usePrevious'
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
 import { isSupportedChainId } from 'lib/hooks/routing/clientSideSmartOrderRouter'
 
+function areCurrenciesTheSame(tradeStateFromUrl: TradeState): boolean {
+  if (!tradeStateFromUrl.inputCurrencyId && !tradeStateFromUrl.outputCurrencyId) return false
```

Comment:
ubernit: destructuring the params just make the statements smaller and slightly easier to read

Reply:
Thanks! Fixed